test/extensions/filters/network/zookeeper_proxy/config_test.cc

```diff
@@ -60,6 +145,139 @@ stat_prefix: test_prefix
   cb(connection);
 }
 
+TEST(ZookeeperFilterConfigTest, ConfigWithDefaultLatencyThreshold) {
+  const std::string yaml = R"EOF(
+stat_prefix: test_prefix
+default_latency_threshold: "0.15s"
+  )EOF";
+
+  ZooKeeperProxyProtoConfig proto_config;
+  TestUtility::loadFromYamlAndValidate(yaml, proto_config);
+
+  testing::NiceMock<Server::Configuration::MockFactoryContext> context;
+  ZooKeeperConfigFactory factory;
+
+  Network::FilterFactoryCb cb = factory.createFilterFactoryFromProto(proto_config, context);
+  Network::MockConnection connection;
+  EXPECT_CALL(connection, addFilter(_));
+  cb(connection);
```

Comment:
should we be checking that the latency threshold made it to the actual config object?